domains/prcv1n.json

```
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "prncvin",
+    "email": "midnightbssyt@gmail.com",
+    "name": "prince",
+    "discord": "@prcv1n"
+  },
+  "records": {
+    "CNAME": "prncvin.github.io/prnc"
```

CNAME doesnt allow any trailing slashes (/)
```suggestion
    "CNAME": "prncvin.github.io"
```

---

i fixed it now

---

> CNAME doesnt allow any trailing slashes (/)

@prncvin just accept this and your good to go